app/views/admin/_space_changes.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="relative bg-gray-100 shadow rounded p-2">
+  <% unless @current_page == 1 %>
+    <%= link_to :back,
+      admin_path(id: @space.id, page: @current_page.to_i - 1),
+      class: "absolute right-16 top-0 p-2 bg-lnu-pink text-white" %>
+  <% end %>
+
+  <%= link_to :next,
+    admin_path(id: @space.id, page: @current_page.to_i + 1),
+    class: "absolute right-0 top-0 p-2 bg-lnu-pink text-white" %>
```

Comment:
Denne sjekker ikke om det finnes en neste side eller ei. Det kunne den sikkert gjort, men trengs ikke for denne siden (den kan være litt dum). Men kanskje legge igjen en kommentar i koden om det?